docs/faster.md

```diff
@@ -469,7 +469,7 @@ After modifying the Rescue-Prime signature scheme to use the new, `FastStark` cl
  - signing time: **72 seconds**
  - verification time: **8 seconds**
 
-How's that for an improvement? The proof is larger because there are many more Merkle paths associated with zerofier leafs, but in exchange verifying is an order of magnitude faster. Of course there is no shortage of further improvements, but those are beyond the scope of this tutorial and left as exercises to the reader.
+How's that for an improvement? The proof is larger because there are many more Merkle paths associated with zerofier leafs, but in exchange for verification that is an order of magnitude faster. Of course there is no shortage of further improvements, but those are beyond the scope of this tutorial and left as exercises to the reader.
```

Comment:
```suggestion
How's that for an improvement? The proof is larger because there are many more Merkle paths associated with zerofier leafs, but in exchange for this larger proof, verification is an order of magnitude faster. Of course there is no shortage of further improvements, but those are beyond the scope of this tutorial and left as exercises to the reader.
```